iocBoot/templates/st.cmd.template

```diff
@@ -81,13 +81,13 @@ dbLoadRecords("{{ db.file }}", "{{ db.macros }}")
 {% endfor %}
 cd "$(TOP)"
 
-dbLoadRecords("db/iocAdmin.db", "P={{prefix}},IOC={{prefix}}" )
-dbLoadRecords("db/save_restoreStatus.db", "P={{prefix}},IOC={{name}}" )
+dbLoadRecords("db/iocAdmin.db", "P={{prefix}}{{delim}},IOC={{prefix}}{{delim}}" )
+dbLoadRecords("db/save_restoreStatus.db", "P={{prefix}}{{delim}},IOC={{name}}" )
```

Comment:
Also, we never compared this file to the edits I made in the st.cmd file. I had to change stuff in both of these lines. Mine look like:
```
dbLoadRecords("db/iocSoft.db", "IOC=IOC:TST:PPM" )
dbLoadRecords("db/save_restoreStatus.db", "P=IOC:TST:PPM:" )
```

Reply:
I think the `:` delimiter might live inside of the admin records...

Reply:
Oh, so one requires the delimiter and one doesn't? That's... annoying.

Reply:
Confirmed. I should have double-checked these first - sorry:

iocAdmin.db: `record(mbbo, "$(IOC):ACCESS")`
save_restoreStatus.db: `record(bo, "$(P)SR_heartbeat")`

Reply:
Yeah I agree, it's super annoying. Also, I'm not sure why the built ioc has `iocSoft.db` instead of `iocAdmin.db`...

Reply:
I think this one's resolved now.